backend/package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This `package.json` is necessary for TypeScript to correctly resolve the path to `types`